client/packages/common/src/hooks/useEditModal/useEditModal.ts

```diff
@@ -20,20 +20,17 @@ export const useEditModal = <T>(): EditModalState<T> => {
   const [entity, setEntity] = useState<T | null>(null);
   const [mode, setMode] = useState<ModalMode | null>(null);
 
-  const onOpen = useCallback(
-    (entity: T | null = null) => {
-      setEntity(entity);
-      setMode(entity ? ModalMode.Update : ModalMode.Create);
-      modalControl.toggleOn();
-    },
-    [modalControl]
-  );
+  const onOpen = (entity: T | null = null) => {
+    setEntity(entity);
+    setMode(entity ? ModalMode.Update : ModalMode.Create);
+    modalControl.toggleOn();
+  };
```

Comment:
Are you sure you can remove this useCallback? this might trigger some other problems of useEditModal users...

The modalControl object is also not cached only the members... So this useCallback should probably better depend on toggleOn and not on the wrapping modalControl object...

Might miss why you did this change.

Reply:
Had discussion on telegram, will merge now since changes have been reverted